.vscode/settings.json

```diff
@@ -16,5 +16,11 @@
     "out": true
   },
   // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
-  "typescript.tsc.autoDetect": "off"
+  "typescript.tsc.autoDetect": "off",
+  "mdb.presetConnections": [
+    {
+      "name": "Preset Database",
```

Comment:
Should this be a "Preset Connection"? It is a bit confusing with the database name, especially when the connection string doesn't include a default database.